### ISSUE TITLE
Analytics bug

### ISSUE THREAD

**Issue by etoweb** [UnamSanctam/UnamWebPanel#265]
![image](https://github.com/UnamSanctam/UnamWebPanel/assets/132036873/42811e5d-c60a-49fe-bc52-de4c3d9dbcb5)

**Comment by etoweb** [comment]
![image](https://github.com/UnamSanctam/UnamWebPanel/assets/132036873/703e8dc5-5ef9-4f66-9cf8-c9038cdee0bb)

**Comment by UnamSanctam** [comment]
Try using the release: https://github.com/UnamSanctam/UnamWebPanel/releases, if you are using the code from the repository directly then there is one line that needs to be changed.

**Comment by etoweb** [comment]
Hello analytics is set on offline, but I have active workers. Any solution ?
I will have soon a lot of workers and i would like know how to repair statistics.
Thanks

**Comment by etoweb** [comment]
> Try using the release: https://github.com/UnamSanctam/UnamWebPanel/releases, if you are using the code from the repository directly then there is one line that needs to be changed.

Thanks i willl try

**Comment by UnamSanctam** [comment]
You only need to replace the `internalPages/statisticsAjax.php` since that's the only different file.

**Comment by etoweb** [comment]
![image](https://github.com/UnamSanctam/UnamWebPanel/assets/132036873/218b32c6-09a4-459e-bff5-69aa8c7ec354)
Thanks :)

**Comment by UnamSanctam** [comment]
No problem, good luck with your mining.

**Comment by etoweb** [comment]
![image](https://github.com/UnamSanctam/UnamWebPanel/assets/132036873/bebd0c72-34ed-4c7f-af08-b3e9205b9bde)
On 2miners I see only one workers, but i have like 5 workers for xmr, are they all count like one workers ?, how can i do to have different workers, with the desktop name of every workers on xmr ?

**Comment by etoweb** [comment]
{
    "algo": "rx/0",
    "pool": "xmr.2miners.com",
    "port": 12222,
    "wallet": "486rV6Rq53DNVJRic8sgDXR7htAdbCJ7mER7wEf4k3XnhmRNFb5Xusd*********************************",
    "password": "",
    "nicehash": false,
    "ssltls": true,
    "max-cpu": 50,
    "idle-wait": 5,
}
It's my remote config, do i need to add after the wallet adress .{COMPUTERNAME} ?

**Comment by UnamSanctam** [comment]
Yes if you want separate workers for each computer. When miners have the same worker name (or no worker name) it will show the combined hashrate of all the miners mining under that worker name.

**Comment by etoweb** [comment]
Maybe that most secure for don't be banned by the pool ? 
Thanks anyway

**Comment by UnamSanctam** [comment]
Yes it's possible, I believe having more workers increases the risk of a ban mostly because it increases the amount of data they have to handle.

**Comment by etoweb** [comment]
sorry again but is it possible to have more H/s on the web panel than in the pool? It's a problem or just normal?

**Comment by UnamSanctam** [comment]
Yes, the web panel gets the hashrate directly from the miner, the pool calculates the hashrate based upon how many shares the miners have submitted over an interval (https://github.com/UnamSanctam/SilentCryptoMiner/wiki#why-is-the-hashrate-shown-on-the-pool-different-from-what-i-expect-or-from-what-is-shown-in-the-web-panel).

**Comment by etoweb** [comment]
> Yes, the web panel gets the hashrate directly from the miners, the pool calculates the hashrate based upon how many shares the miners have submitted over an interval (https://github.com/UnamSanctam/SilentCryptoMiner/wiki#why-is-the-hashrate-shown-on-the-pool-different-from-what-i-expect-or-from-what-is-shown-in-the-web-panel).

thanks :)
